go/hyperkit.go

```diff
@@ -486,8 +485,9 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	log.Debugf("hyperkit: CmdLine: %#v", h.CmdLine)
 }
 
-// Execute hyperkit and plumb stdin/stdout/stderr.
-func (h *HyperKit) execHyperKit() error {
+// makeCommand forges the command to run hyperkit, runs and returns it.
+// It also plumbs stdin/stdout/stderr.
+func (h *HyperKit) makeCommand() (*exec.Cmd, error) {
```

Comment:
`makeCommand` to me suggests it constructs the `Command` object but not that it runs it. I think I see why you ant to avoid `exec` and `forkAndExec` is a bit of a mouthful. How about `launchCommand`?

Reply:
Or given how it is used how about just `launch` (caller would be `h.launch()`).

Reply:
I was going patch by patch and the last one resolves this! I should have used the GH review mode instead of comment by comment so I could catch myself...

Reply:
Hi Ian!

Yes, it also happens to me a lot :)  In that case I something just remove my comment.